### DISCUSSION TITLE
Anthem MRF data incorrect

### DISCUSSION THREAD

**Discussion by r-shelby** [CMSgov/price-transparency-guide#672] [Q&A]
We've noticed some issues in data quality with Anthem INR MRFs located here - https://www.anthem.com/machine-readable-file/search/

For the same plan, holding NPI constant, we're seeing the same rates across multiple billing codes. Some are blatantly wrong (like $20K for a simple drainage and the same rate for vaginal delivery). Is anyone experiencing the same issues? Attaching a snippet for reference (EIN: 77-0162263).
[anthem_ukiah_codes.xlsx](https://github.com/CMSgov/price-transparency-guide/files/11368739/anthem_ukiah_codes.xlsx)

**Comment by dgolden** [comment]
@r-shelby Using our free online search tool MRF Explorer, I took a peek at a May Anthem INR MRF and didn't see the issue you encountered. I suggest that you plug-in the URL for the MRF in question into our tool, and compare the results.

https://www.keyark.xyz/test/mrfreporter/mrfreportapp

I'm available for a follow-up discussion. See my GitHub profile for contact information.

**Reply by r-shelby** [reply]
Thanks for sharing. I checked with the tool and seeing the same issue - here's a screenshot of the report generated from the tool for that NPI.
![image](https://user-images.githubusercontent.com/114793533/235712859-ac7a244f-fde5-4a05-b7fd-4384e7502c86.png)

**Reply by dgolden** [reply]
@r-shelby I think it is safe to conclude that Anthem MRF production is broken. From what I see, and hear on the grapevine, payers are not putting a lot of effort into QA. Per a recent CMS press release*, enforcement of hospital price transparency is gearing up. Maybe enforcement of health plan MRF transparency will follow soon.

As a stop gap measure, given the likely overlap in data among MRFs from the same payer, perhaps you can accomplish your goal by ignoring the flawed MRFs and extracting the data from other MRF that were correctly generated. 

In a perfect world, the MRF spec would include contact information so you could contact the payer directly. I had success reaching out directly to one Blue, but I got lucky in knowing the contact there. The general recommendation is to file a complaint with CMS via their web form: https://www.cms.gov/healthplan-price-transparency/contact-us

*https://www.beckershospitalreview.com/finance/cms-makes-changes-to-hospital-price-transparency-enforcement-process.html

**Reply by brendanyounger** [reply]
@r-shelby Which file from Anthem? I'm currently processing their data as well. And I'm trying to keep a record of all the errors I come across here https://github.com/tynbil/transparency-in-coverage/blob/main/year%3D2023/month%3D05/payer%3Danthem/notes.md 

For the CMS folks, is there a central place we can log issues we find when processing the MRFs? We're all dealing with it in different ways and it'd be nice to have some way of holding the payers accountable.

**Reply by r-shelby** [reply]
Thanks @dgolden, appreciate your quick response. 

@brendanyounger - for EIN: 77-0162263, all CA INR files have this error so far, but we haven't unpacked beyond that. I like your idea of creating a central log - would you be open to co-creating one?

**Reply by brendanyounger** [reply]
@r-shelby Appreciate the pointer to the bad data. I'd definitely be interested in collaborating on a central database of data quality issues. Maybe a Github repo? My email is brendan@tynbil.com

**Reply by santaviccap** [reply]
Has anyone here who's worked with Anthem had any luck identifying their network names in their files?  I see a ton of data where the only descriptor is something like MO_OSBHMED0000 and haven't been able to tell what network it is.

**Reply by BT-audreymalek** [reply]
@santaviccap -- we have found that the first part of the string seems to align with the state, so in this case Missouri... a google search for "Missouri Blue Cross OSBA" returns this which may point you in the right direction! https://capstoneins.com/osba-program/interested-districts/

**Comment by dgolden** [comment]
@r-shelby @brendanyounger  For your project idea, you may want to network with contact@tictoc.health ( see https://tictoc.health/ ). Somewhere in this github discussion group 'tictoc' posted their inventory of issues encountered. The URL has the same (possibly updated) data.

@r-shelby I examined only a single Anthem CA INR MRF, and the data looked okay. Here is a screenshot, with a box denoting the one I checked. Happy to chat further, but best to switch to email as our discussion group message traffic is relayed to 250+ mail inboxes.

![anthem_mrf](https://user-images.githubusercontent.com/6385204/235751509-939deaa6-ae55-4e8b-827a-778a2ed0fd7e.jpg)

**Reply by sudhasridhar** [reply]
We are having same issues with anthem files as well, one example https://antm-pt-prod-dataz-nogbd-nophi-us-east1.s3.amazonaws.com/anthem/CA_BCCSMED0220_02_04.json.gz price for diagnostic colonoscopy 45378 @$41,138, almost like prices are stated in cents.

**Comment by AshleyOboe** [comment]
Just a couple of thoughts from what I've seen:

- The institutional codes seem like facility fees, which can be 10-20k per stay.  These wouldn't include the professional fees for the services. This probably doesn't explain all of the high values, but I wouldn't be surprised if they were negotiated in bulk (at least the base rates).
- I've seen some entries where there is a high cost as well as a percentage with the same provider_id_group in the file.  I believe what this means is that the provider is reimbursed a percentage of the fee schedule amount. 
- I've been told that complex institutional procedure costs (i.e. back surgery) will be difficult to interpret from this data because of the way that hospitals carve out various acuity levels and terms in their negotiated rates.  What we are seeing is probably the base rate that then has some adjustments based on case.  While we would have hoped some of that is explained in the fields for additional information, it probably is nestled into obscure contract and database systems that would be difficult to pull into the datasets.

Just a few theories.  

And for what it's worth @r-shelby  - the BCBS CA data looks a little more reasonable for this provider, with a lot of the institutional rates between 1k and 13k.

Thanks for everyone's contributions to sorting it out.
